### PR TITLE
[Th2-5165] Use cache in BookInfo

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: cradle-admin-tool-cli/src/test/java/com/exactpro/th2/cradle/adm/cli/BookCliTest.java [hunk 1]
```diff
@@ -12,103 +12,109 @@
...
-                            Assertions.assertNotNull(dev_test_5);
-                            Assertions.assertNotNull(dev_test_5.getCreated());
-                            Assertions.assertTrue(dev_test_5.getCreated().isAfter(i1));
-                            Assertions.assertTrue(dev_test_5.getCreated().isBefore(Instant.now()));
-
-                            checkOutput(true, null);
-                        }
-                );
+    public void addBookWithoutTimeTest(@Th2AppFactory CommonFactory appFactory,
+                                       @Th2TestFactory CommonFactory testFactory) throws Exception {
```
[the remark this answers is not in the document]
Comment (Nikita-Smirnov-Exactpro) [reply cradle-admin-tool-cli/src/test/java/com/exactpro/th2/cradle/adm/cli/BookCliTest.java:57]: How can I do that?

[the remark this answers is not in the document]
Comment (OptimumCode) [reply cradle-admin-tool-cli/src/test/java/com/exactpro/th2/cradle/adm/cli/BookCliTest.java:57]: Like you did in `initStorage` method...

File: cradle-admin-tool-http/build.gradle [hunk 2]
```diff
@@ -27,10 +27,6 @@ application {
     mainClassName = "com.exactpro.th2.cradle.adm.http.Application"
 }
 
-test {
-    useJUnitPlatform()
-}
```
Comment (OptimumCode) [inline cradle-admin-tool-http/build.gradle:32]: Could you let me know why you had to remove it?

Comment (OptimumCode) [reply cradle-admin-tool-http/build.gradle:32]: Got it. Missed that moment

Comment (Nikita-Smirnov-Exactpro) [reply cradle-admin-tool-http/build.gradle:32]: No, problem )

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**